flink-table/flink-sql-gateway/src/test/resources/sql/catalog_database.q

```diff
@@ -911,3 +911,35 @@ desc catalog extended cat2;
 +-------------------------+-------------------+
 4 rows in set
 !ok
+
+alter catalog cat2 reset ('default-database', 'k1');
```

Comment:
Nit: I found that a section comment called `# test catalog` already exists at L#47, and it'd be better to gather all catalog-related tests together to improve readability.

Reply:
done :)